### PR TITLE
Adding new page for Built-In Toolbar 

### PR DESCRIPTION
**Pull request by Bharathi4332**
Title : Create a UG Document for Built-In Toolbar 
Task link : https://dev.azure.com/EssentialStudio/Mobile%20and%20Desktop/_sprints/backlog/PdfViewerMAUI/Mobile%20and%20Desktop/2024Jun1_PDFViewer_MAUI?workitem=890255

### CODE REVIEW

**Comment by kannans24** [comment]
rebuild

File: MAUI/PDF-Viewer/Built-In-Toolbar.md [hunk 1]
```diff
@@ -0,0 +1,741 @@
...
+description: Learn here all about the built-in toolbar in .NET MAUI PDF Viewer (SfPdfViewer) control, its functionalites and more.
+platform: MAUI
+control: SfPdfViewer
+documentation: ug
+keywords : .net maui pdf viewer, .net maui view pdf, pdf viewer in .net maui, .net maui open pdf, maui pdf viewer, maui pdf view
+---
+
+# Working with the Built-In Toolbar in .NET MAUI Pdf Viewer (SfPdfViewer)
+
+The [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html) has a built-in toolbar, which allows you to perform operations such as adding, modifying annotations, searching text, etc. Additionally, you have the option to show or hide the default built-in toolbar and its sub-toolbars and toolbar items, such as buttons, and you can customize your own toolbar by disabling the built-in toolbar and implementing one at the application and hooking the custom toolbar UI to the PDF viewer APIs. You can see the custom toolbar example [here](https://github.com/SyncfusionExamples/maui-pdf-viewer-examples/tree/master/Custom%20Toolbar).
```
Comment (Deepak1799) [inline MAUI/PDF-Viewer/Built-In-Toolbar.md:13]: image

Use different word for this

File: MAUI/PDF-Viewer/Built-In-Toolbar.md [hunk 2]
```diff
@@ -0,0 +1,741 @@
...
+
+  </ContentPage.Content>
+{% endhighlight %}
+{% endtabs %}
+
+## Customize the Toolbar 
+
+To customize the toolbar, you can programmatically add, remove, and hide the items based on your needs.
+
+### Customizing toolbar items using the GetByName method
```
Comment (Deepak1799) [inline MAUI/PDF-Viewer/Built-In-Toolbar.md:44]: Why continuously 3 header types are used?

File: MAUI/PDF-Viewer/Built-In-Toolbar.md [hunk 3]
```diff
@@ -0,0 +1,741 @@
...
+
+var item = pdfViewer.Toolbars?.GetByName("PrimaryToolbar")?.Items?.GetByName("Search");
+if (item != null)
+{
+item.IsVisible = false; // Hide the search item
+}
+{% endhighlight %}
+{% endtabs %}
+
+### Adding a New Toolbar Item
```
Comment (Deepak1799) [inline MAUI/PDF-Viewer/Built-In-Toolbar.md:62]: **### Adding a New Toolbar Item**

In contents header, only the first letter of the first word should be in **caps**, others are need to be in the small casing.

**Ensure address this in all the areas**

image

File: MAUI/PDF-Viewer/Built-In-Toolbar.md [hunk 4]
```diff
@@ -0,0 +1,741 @@
...
+
+To add an item to the toolbar in [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html), first, create the button you want to include. Then, convert that button into a ToolbarItem using the `ToolbarItem` method. Finally, add the newly created ToolbarItem to the toolbar using the `Add` method. Here we create the new button fileOpenButton and retrieve the "PrimaryToolbar" using the GetByName method. Add the new button to the "Primary Toolbar".
```
Comment (Deepak1799) [inline MAUI/PDF-Viewer/Built-In-Toolbar.md:64]: Generalise the item or say it as an example.

image

In the above contents, it conveys only button can be added as item.

File: MAUI/PDF-Viewer/Built-In-Toolbar.md [hunk 5]
```diff
@@ -0,0 +1,741 @@
...
+ if (item != null)
+ {
+     pdfViewer.Toolbars?.GetByName("PrimaryToolbar")?.Items?.Remove(item); // Remove the outline item
+ }
+{% endhighlight %}
+{% endtabs %}
+
+Similarly, you can customize the other toolbars and their items by accessing their names. We have listed the other toolbar and toolbar item names below:
+
+### Mobile Toolbar Name :
```
Comment (Deepak1799) [inline MAUI/PDF-Viewer/Built-In-Toolbar.md:139]: **### Mobile Toolbar Name**

It should be mentioned in plural. Please correct

**Review by Deepak1799 — CHANGES_REQUESTED** [review]
Requested changes

**Comment by Bharathi4332** [comment]
I have addressed the feedback

File: MAUI/PDF-Viewer/Toolbar.md [hunk 6]
```diff
@@ -0,0 +1,741 @@
...
+control: SfPdfViewer
+documentation: ug
+keywords : .net maui pdf viewer, .net maui view pdf, pdf viewer in .net maui, .net maui open pdf, maui pdf viewer, maui pdf view
+---
+
+# Working with Toolbar in .NET MAUI Pdf Viewer (SfPdfViewer)
+
+The [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html) has a built-in toolbar, which allows you to perform operations such as adding, modifying annotations, searching text, etc. Additionally, you have the option to show or hide the default built-in toolbar, its sub-toolbars, and toolbar items. You can customize your own toolbar by disabling the built-in toolbar, then implementing one in the application and integrating the custom toolbar UI into the PDF viewer APIs. You can see the custom toolbar example [here](https://github.com/SyncfusionExamples/maui-pdf-viewer-examples/tree/master/Custom%20Toolbar).
+
+## Show/hide the built-in toolbar
```
Comment (VikramDevaraj31) [inline MAUI/PDF-Viewer/Toolbar.md:15]: Built-In Toolbar => Toolbar

**Review by Deepak1799 — APPROVED** [review]
Approved the changes

**Pull request merged**